README.md

```diff
@@ -304,7 +304,7 @@ Please read: https://chris.beams.io/posts/git-commit/
 - Always check your commit in details to avoid committing wrong code.
 
 ### Code Reviews
-Always let someone else review your code in the Pull/Merge Request. Make sure all code review comments are resolved, before you merge it!
+Always let someone else review your code in the Pull/Merge Request. Make sure all code review comments are resolved, before you merge it! Read the complete guidelines [here](https://github.com/mediamonks/frontend-coding-standards/blob/main/creating-pull-requests.md).
```

Comment:
I don't like `here` links. They are not user (and SEO) friendly.  Can we change this to:
```suggestion
Always let someone else review your code in the Pull/Merge Request. Make sure all code review comments are resolved, before you merge it! Please read our [Pull Request Guidelines](https://github.com/mediamonks/frontend-coding-standards/blob/main/creating-pull-requests.md).
```